mongo-client/Dockerfile

```diff
@@ -1,35 +1,29 @@
 FROM ubuntu:16.04
 
-RUN apt-get clean \
+RUN apt-get update && apt-get -y --force-yes install \
```

Comment:
Thanks for the changes @bilal-mk. This changes seems to be good but we tested this Docker file in our own environment and it doesn't work for the purpose. Can you kindly push the changes with the following Docker spec:
```
FROM ubuntu:16.04

RUN apt-get update -y || true

RUN apt-get install -y --force-yes --no-install-recommends \
    python \
    python-pip \
    make \
    automake \
    libmysqlclient-dev \
    libtool \
    libsasl2-dev \
    libssl-dev \
    libmongoc-dev \
    libbson-dev

RUN apt-get clean \
  && rm -rf /var/lib/apt/lists/* \
  && cp /usr/include/libbson-1.0/* /usr/include/ \
  && cp /usr/include/libmongoc-1.0/* /usr/include/
#  && pip install --upgrade pip 

RUN pip install --upgrade setuptools
RUN  python -m pip install pystrich \
  && python -m pip install pymongo

ADD sysbench-mongo/sysbench /sysbench
WORKDIR /sysbench
RUN ./autogen.sh && ./configure && make

# components for liveness script
ADD liveness/server.py ./
```

Reply:
Same is this pull request with reduced the few-layer as part of the Dockerfile best practice. Please share how to properly test this or add those test as part of travis integration test.

Reply:
Agreed @bilal-mk. Have raised #49 & #50 to track this.

Currently travis performs a build check alone for all the images and doesn't look for existence of packages/artifacts specified in the Dockerfile (saw a few commits pushed as "fixes" for those that broke the build), which is what has necessitated certain manual tests. One way of testing this is to look at the python/bash scripts being pushed into the image & verifying the presence of the binaries &/or auxiliary files used in it in the locally built image. 

Amongst the multiple docker images maintained in this repo & litmus - hacktoberfest has helped us refactor the images & its CI to the "extent possible", i.e., w/o breaking prod deployments where they are being used as K8s resources. Of course, further optimizations are possible by refactoring the scripts used in them - but these will be taken up based on priority.

Reply:
In this particular case, the location where apt-get clean is specified seems to sporadically cause unsuccessful install of some packages.

Reply:
The travis build has passed(https://api.travis-ci.org/v3/job/443697398/log.txt) & I have tested this on a GCP compute instance (you can guarantee a clean build env here!).  So, we should be good ot go!